CDCgov/prime-reportstream
# Update org file for  CA

This PR updates one of the receivers for CA in the local org file, to match what is in prod.

Test Steps:
1. generate fake data
2. fire up docker
3. reloadsettings
4. send fake data to api
5. save the generated file
6. pull settings from prod env
7. copy and paste the prod env receiver settings to replace the settings for the branch. Do not replace the transport, leave that alone.
8. reloadsettings
9. send fake data to api
10. compare generated file with the other one, make sure they are the same

## Changes
- updated the prod feed for CA in the local org file
